galsim/phase_screens.py

```diff
@@ -0,0 +1,623 @@
+import numpy as np
+import galsim
+import utilities
+
+class AtmosphericScreen(object):
+    """ An atmospheric phase screen that can drift in the wind and evolves ("boils") over time.  The
+    initial phases and fractional phase updates are drawn from a von Karman power spectrum, which is
+    defined by a Fried parameter that effectively sets the amplitude of the turbulence, and an outer
+    scale that sets the scale beyond which the turbulence power goes (smoothly) to zero.
```

Comment:
...and an outer scale beyond which...?
(seems like the extra words are not needed)